### ISSUE TITLE
Drop onto Canvas

### ISSUE THREAD

**Issue by Larkinabout** [Larkinabout/fvtt-challenge-tracker#8]
Allow Challenge Trackers to be dropped onto the canvas as a token element. Double-click or some other function to pick up (open in application window). See if completing segments while displayed as a token element is also possible.

This one will take a while.

**Comment by brunocalado** [comment]
I don't know if this worth. The clocks (same feature) have lots of problems. I replace their use for your module.

https://github.com/mattd/foundryvtt-progress-clocks

**Comment by Larkinabout** [comment]
Good point. It's also probably not as necessary now with the challenge tracker UI.

I'll let this one chill and see if anyone else really wants it or if I get desperate for it myself. In either case I'd do it as an addition, so you can display it either way.

**Comment by w1face** [comment]
Would love this option. Ability to have two or more trackers open at the same time, and locked onto the canvas would be amazing (See example)

![Image](https://github.com/user-attachments/assets/db5cdd41-0d34-4988-93ab-3b29757ae5c7)
